apps/user-portal/src/locales/ta/views.ts

```diff
@@ -1010,13 +1010,18 @@ export const views: Views = {
         security: {
             subTitle: "உங்களின் கணக்கின் பாதுகாப்பை உறுதிப்படுத்த அமைப்புக்களை புதுப்பிக",
             title: "பாதுகாப்பு"
-        },
-        404: {
-            subTitle: "நீங்கள் தேடி வந்த பக்கம் இங்கில்லை.",
-            title: "நீங்கள் வழி தவறி வந்து விட்டதாக தெரிகிறது :("
         }
     },
     placeholders: {
+        404: {
```

Comment:
@thivi Please validate this 404 block as well

Reply:
பக்கத்தின் முகவரியினை சரி பாருங்கள் அல்லது கீழிருக்கும் பொத்தானை அழுத்தி